### PR TITLE
ML for Trading lab - course 2

### PR DESCRIPTION
**Pull request by priumoraes**
Lab on Intro to tf.data and Keras Sequential API

### CODE REVIEW

File: courses/ai-for-finance/practice/intro_tf_data_keras_sequential.ipynb [hunk 1]
```diff
@@ -0,0 +1,937 @@
+{
```
Comment (gstripling) [inline courses/ai-for-finance/practice/intro_tf_data_keras_sequential.ipynb:1]: <p>Would suggest:</p><p> </p><p><strong>Lab Scope</strong>:  This lab does not cover how to make predictions on the model or deploy it to Cloud AI Platform.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (gstripling) [inline courses/ai-for-finance/practice/intro_tf_data_keras_sequential.ipynb:1]: <p>We typically enumerate learning objectives, so that the TODO's can mirror student activities. We don't have a standard, but ASL (and TCD's) use enumeration.</p><p> </p><ol><li>Apply techniques to clean and inspect data.</li><li>Split dataset into training, validation and test datasets.</li><li>Use the tf.data.Dataset to create an input pipeline.</li><li>Define, compile and train a model using the Keras Sequential API.</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (gstripling) [inline courses/ai-for-finance/practice/intro_tf_data_keras_sequential.ipynb:1]: <p>Suggest bold <strong>Note. </strong>In Markdown - two asterisks on either side.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (gstripling) [inline courses/ai-for-finance/practice/intro_tf_data_keras_sequential.ipynb:1]: <p>Last sentence - replace "of" with "we", ...do you see any issues with the examples we have to train on?  </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (gstripling) [inline courses/ai-for-finance/practice/intro_tf_data_keras_sequential.ipynb:1]: <p>First sentence, "test sets" should be "test set".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (gstripling) [inline courses/ai-for-finance/practice/intro_tf_data_keras_sequential.ipynb:1]: <p>Add a parentheses after label.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (gstripling) [inline courses/ai-for-finance/practice/intro_tf_data_keras_sequential.ipynb:1]: <p>First sentence, first paragraph - change "&nbsp;creare"  to "creatre.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (priumoraes) [reply courses/ai-for-finance/practice/intro_tf_data_keras_sequential.ipynb:1]: thanks!

**Comment by priumoraes** [comment]
Thanks Gwendolyn!
